### ISSUE TITLE
It is not working on android

### ISSUE THREAD

**Issue by cngeru** [leecade/react-native-swiper#777]
### Which OS ?
Windows
### Version
10
Which versions are you using:

- react-native-swiper v1.5.13
- react-native v0.55

### Expected behaviour
Expect to load a sliding carousel on android
### Actual behaviour
displays this weird screen
![screenshot_20180503-142443](https://user-images.githubusercontent.com/21263893/39573956-f562d0de-4edd-11e8-8dd8-2748ce48fcf8.png)
### How to reproduce it-
`import React from "react";
import {
    Dimensions,
    View,
    StyleSheet,
    Text,
    Platform
} from "react-native";
import Swiper from "react-native-swiper";

class Splash extends React.Component {
  render() {
        return (
            <View style={styles.container}>
              <View style={{height: 200,alignItems:'flex-start'}}> 
               <Swiper
                 showsButtons={false}
                 loop={false}
                 >
                <View style={styles.container}>
                  <Text>
                    Fun
                  </Text>
                </View>
                <View style={styles.container}>
                  <Text>
                    play,chat,have fun
                  </Text>
                </View>
              </Swiper>
             </View>
            </View>
        );
    }
}
export default Splash;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});`
### Steps to reproduce
Try running on expo

**Comment by yaohb** [comment]
i have the same problem, it's not working for android!!

**Comment by alangumer** [comment]
Same here, is not working on Android, react-native 0.55.3

**Comment by wesleymonaro** [comment]
Same here, using react-native 0.54.4. Tried downgrade to swipe version 1.5.12, without success :/

In IOS emulator works fine.

**Comment by maks171293** [comment]
Also have the same error. Looks like this is because of react-navigation. If I put the same code to the new project without react-navigation, the content is appear.

**Comment by swingywc** [comment]
Same here using react-native `v0.52.0`, one weird thing is when I am using Hot Reloading and make some changes, the content will appear.

**Comment by booleanBoy** [comment]
Same here with the same setup as @swingywc on Android. I've got the pagination indicators, but not the page contents.

**Comment by alangumer** [comment]
It looks like the issue has something to do with the ViewPagerAndroid, take a look at this [PR](https://github.com/leecade/react-native-swiper/pull/761/commits/1cf7352d3a18a1d63a42712778285cafaa0778e9)

**Comment by booleanBoy** [comment]
Just solve the issue for myself, which is related to the Modal I have wrapped around it. See my comment on #435 for example code.

**Comment by booleanBoy** [comment]
Looked at the PR @alangumer and it works just fine, thanks!

**Comment by CricketLaChica** [comment]
I've tried doing the PR by @alangumer, which basically bypasses ViewPagerAndroid and use ScrollView – the issue is, it doesn't support things like going to a specific index for Android. Did this use to work for Android before?

**Comment by stevula** [comment]
For me it was not working when `<Swiper>`'s parent had `alignItems: 'center'`. It worked fine when I removed that property. If it's relevant, the parent was an `<ImageBackground>` element with the following styles:

```
{
    flex: 1,
    justifyContent: 'flex-start',
    alignItems: 'center',
  }
```

It worked fine when I changed the parent styles to this:
```
{
    height: '100%',
    width: '100%',
}
```

But it also worked when I just commented out the `alignItems` rule I had originally.

**Comment by kyunkakata** [comment]
@stevula you are my life savior. Thank you. ^^

**Comment by vicros0723** [comment]
@stevula removing `alignItems: 'center'` worked for me when the parent was a `<View>` element. Amazing catch.

**Comment by almorak** [comment]
[ android ] not work in a modal + parent view, even no alignItems.
*using react-navigation

**Comment by halbano** [comment]
Hey @stevula, lifesaver finding. What is your theory about the issue? Would you mind sharing it with me? Thanks in advance.

**Comment by anshgupta1234** [comment]
> For me it was not working when `<Swiper>`'s parent had `alignItems: 'center'`. It worked fine when I removed that property. If it's relevant, the parent was an `<ImageBackground>` element with the following styles:
> 
> ```
> {
>     flex: 1,
>     justifyContent: 'flex-start',
>     alignItems: 'center',
>   }
> ```
> 
> It worked fine when I changed the parent styles to this:
> 
> ```
> {
>     height: '100%',
>     width: '100%',
> }
> ```
> 
> But it also worked when I just commented out the `alignItems` rule I had originally.
> 
> **TLDR: just remove `alignItems: 'center'` from the parent element.**

I'm forever indebted to you. Thanks a lot

**Comment by navanit-jha** [comment]
> For me it was not working when `<Swiper>`'s parent had `alignItems: 'center'`. It worked fine when I removed that property. If it's relevant, the parent was an `<ImageBackground>` element with the following styles:
> 
> ```
> {
>     flex: 1,
>     justifyContent: 'flex-start',
>     alignItems: 'center',
>   }
> ```
> 
> It worked fine when I changed the parent styles to this:
> 
> ```
> {
>     height: '100%',
>     width: '100%',
> }
> ```
> 
> But it also worked when I just commented out the `alignItems` rule I had originally.
> 
> **TLDR: just remove `alignItems: 'center'` from the parent element.**

Thank you so much : )
You are amazing
